EnglishTranslation-master/SKILL.tsv

```diff
@@ -9690,195 +9690,195 @@ SKILL_20171128_009689	Shoot Lightning to enemy	적에게 뇌 탄을 발사합니
-SKILL_20171128_009700	Duration 300 sec{nl}Dragoon skill damage +50%{nl}Dragoon skill SP consumption -50%{nl}Max. level 1	지속시간 300초{nl}드라군 스킬 대미지 50% 증가{nl}드라군 스킬의 SP소모 50% 감소{nl}1레벨 마스터				
-SKILL_20171128_009701	Shares beneficial properties with party members within the range.	일정 범위 내에 있는 파티원들의 이로운 효과를 공유합니다.	SKILL_20161005_006547	Shares beneficial buffs with party members. When sharing buffs, the limits of buffs can be ignored.	일정 범위 내에 있는 파티원들의 이로운 효과를 공유합니다. 버프를 공유받을 때는 버프 제한수를 무시하고 공유받을 수 있습니다.	
-SKILL_20171128_009702	NonInvasive Area	불가침 영역				
-SKILL_20171128_009703	Set the NonInvasive Area around your self. This area gives guild members the ability to block magic.	자신의 주변에 불가침 영역을 설정합니다. 해당영역은 길드원에게 마법을 블럭 시켜주는 효과를 부여해줍니다.				
-SKILL_20171128_009704	Duration #{CaptionTime}#seconds{nl}Additional Block Chance #{CaptionRatio}#%{nl}for 1 seconds SP #{CaptionRatio2}#Consume	지속시간 #{CaptionTime}#초{nl}추가 블럭 확률 #{CaptionRatio}#%{nl}1초 당 SP #{CaptionRatio2}#소비				
-SKILL_20171128_009705	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Fix your grip on the spear and strike your enemy down downward and take the target off of the companion.	{#DD5500}{ol}[물리] - [찌르기]{/}{/}{nl}창을 고쳐잡아 위에서 아래로 찍어내리는 공격으로 컴패니언에 탑승한 적을 바닥으로 떨어뜨립니다. 컴패니언에 탑승한 적, 야수형 적에게 추가대미지를 줍니다.	SKILL_20170420_008490	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Fix your grip on the spear and strike your enemy down downward and take the target off of the companion.	{#DD5500}{ol}[물리] - [찌르기]{/}{/}{nl}창을 고쳐잡아 위에서 아래로 찍어내리는 공격으로 컴패니언에 탑승한 적을 바닥으로 떨어뜨립니다. 야수형 적에게 추가대미지를 줍니다.	
+SKILL_20171128_009699	Equip your Dragoon Helmet. When equip the helmet, all Dragoon skills are enhanced by increasing their damage dealt and reducing SP consumption. Use the skill again to remove the helmet.	드라군 헬멧을 착용합니다. 헬멧을 착용하면 모든 드라군의 스킬이 강화되어 적에게 주는 대미지가 증가하고, SP소모가 감소합니다. 헬멧을 착용한 상태에서 다시 사용하면 헬멧을 벗습니다.				
+SKILL_20171128_009700	Duration 300 sec{nl}Dragoon Skill Damage +50%{nl}Dragoon Skill SP Consumption -50%{nl}Max Level: 1	지속시간 300초{nl}드라군 스킬 대미지 50% 증가{nl}드라군 스킬의 SP소모 50% 감소{nl}1레벨 마스터				
+SKILL_20171128_009701	Shares beneficial properties with party members within range.	일정 범위 내에 있는 파티원들의 이로운 효과를 공유합니다.	SKILL_20161005_006547	Shares beneficial buffs with party members. When sharing buffs, the limits of buffs can be ignored.	일정 범위 내에 있는 파티원들의 이로운 효과를 공유합니다. 버프를 공유받을 때는 버프 제한수를 무시하고 공유받을 수 있습니다.	
+SKILL_20171128_009702	Non-Invasive Area	불가침 영역				
+SKILL_20171128_009703	Create a Non-Invasive Area around yourself. This area gives guild members the ability to block magic.	자신의 주변에 불가침 영역을 설정합니다. 해당영역은 길드원에게 마법을 블럭 시켜주는 효과를 부여해줍니다.				
+SKILL_20171128_009704	Duration #{CaptionTime}#seconds{nl}Additional Block Chance #{CaptionRatio}#%{nl}Consumes #{CaptionRatio2}# SP per second	지속시간 #{CaptionTime}#초{nl}추가 블럭 확률 #{CaptionRatio}#%{nl}1초 당 SP #{CaptionRatio2}#소비				
```

Comment:
`Duration #{CaptionTime}#seconds{nl}Additional Block Chance #{CaptionRatio}#`

to

`Duration: #{CaptionTime}# seconds{nl}Additional Block Chance: #{CaptionRatio}#`